draft-normington-jsonpath-latest.md

```diff
@@ -425,22 +428,25 @@ An array index is a union element consisting of an integer (in base 10).
 ~~~~ abnf
 array-index = integer
 
-integer = [%x2D] (%x30 / (%x31-39 *%x30-39))
+integer = ["-"] ("0" / (%x31-39 *%x30-39))
                             ; optional - followed by 0 or
                             ; sequence of digits with no leading zero
+DIGIT1 = %x31-39            ; non-zero digit
 ~~~~
 
 Note: the syntax does not allow integers with leading zeros such as `01` and `-01`.
 
-An array slice is a union element consisting of two or three integers (in
-base 10 and which may be omitted) separated by colons.
+An array slice is a union element consisting of optional integers (in base 10) separated by colons.
 
 ~~~~ abnf
-array-slice = [ integer ] ws %x3A ws [ integer ]
-                   [ ws %x3A ws [ integer ] ]
-                            ; start:end or start:end:step
+array-slice = [ start ] ws ":" ws [ end ]
+                   [ ws ":" ws [ step ] ]
+start = integer
+end = integer
+step = integer
```

Comment:
Does the markdown translation align the `=`?

Reply:
No. Does that matter?

Reply:
Typically I see them aligned for readability, but technically, no.

Reply:
I suggest we leave that for now and do a fine-tuning editorial pass later for consistency.